lib/active_record/virtual_attributes/virtual_arel.rb

```diff
@@ -22,17 +46,13 @@ module VirtualArel
       end
 
       module ClassMethods
-        def arel_attribute(column_name, arel_table = self.arel_table)
-          load_schema
-          if virtual_attribute?(column_name) && !attribute_alias?(column_name)
-            if (col = _virtual_arel[column_name.to_s])
-              arel = col.call(arel_table)
-              arel.name = column_name if arel.kind_of?(Arel::Nodes::Grouping)
-              arel
-            end
-          else
-            super
-          end
+        # ActiveRecord::Core
+        def arel_table # :nodoc:
+          @arel_table ||= if ActiveRecord.version.to_s < "6.1"
```

Comment:
Super minor, but can we extract the version check to outside of the method, maybe into a constant?  Otherwise we are constant looking up the version and converting to a string.

Reply:
This will be duplicate code.
I agree with the premise. Just hope that there are not nuances that I missed with these changes

Reply:
extracted to another method. duplication be gone